CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 ### Fixed
 - Fixed joint acceleration filtering (See [!124](https://github.com/robotology/whole-body-estimators/pull/124)).
-
+- Fixed `wholeBodyDynamics` device when loaded by the `yarprobotinterface` from an urdf[!126](https://github.com/robotology/whole-body-estimators/issues/126)
```

Comment:
I am not sure this that it is clear what this means, in particular what it means "loaded by the `yarprobotinterface` from an urdf". Perhaps "when loaded by the `gazebo_yarp_robotinterface` Gazebo plugin" could be more clear?